lib/fastlane/actions/set_github_release.rb

```diff
@@ -176,6 +177,15 @@ def self.available_options
                                          raise "Please only pass the path, e.g. 'fastlane/fastlane'".red if value.include? "github.com"
                                          raise "Please only pass the path, e.g. 'fastlane/fastlane'".red if value.split('/').count != 2
                                        end),
+          FastlaneCore::ConfigItem.new(key: :server_url,
+                                       env_name: "FL_GITHUB_RELEASE_SERVER_URL",
+                                       description: "The server url. e.g. 'https://your.internal.github.host/api/v3' (Default: 'https://api.github.com')",
+                                       default_value: 'https://api.github.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.